cfme/containers/image.py

```diff
@@ -22,6 +25,18 @@
 )
 
 
+def run_smart_state_analysis(self):
```

Comment:
Right, this can be run against a single image. I agree about the mixin for SSA, it's just not available right now.